packages/pg-cluster/src/index.ts

```diff
@@ -0,0 +1,9 @@
+import {Queryable} from '@databases/pg';
+import Cluster from './Cluster';
+
+export default function createCluster(
+  primary: Queryable,
+  ...replicas: Queryable[]
+) {
+  return new Cluster(primary, replicas);
+}
```

Comment:
I think the API will be more obvious if we make it take a primary and an array of replicas, rather than using spread parameters. We want it to be obvious that you can have multiple replicas.

```suggestion
export default function createCluster(
  primary: Queryable,
  replicas: Queryable[]
) {
  return new Cluster(primary, replicas);
}
```